opentelemetry/proto/common/v1/common.proto

```diff
@@ -21,6 +21,36 @@ option java_package = "io.opentelemetry.proto.common.v1";
 option java_outer_classname = "CommonProto";
 option go_package = "go.opentelemetry.io/proto/otlp/common/v1";
 
+// inlined parquet-gen options
+import "google/protobuf/descriptor.proto";
+
+message MessageOptions {
+  // As long as table_name is not blank,
+  // a schema is generated for top-level messages in each file.
+  string table_name = 1;
+}
+
+extend google.protobuf.MessageOptions {
+  MessageOptions message_opts = 1023;
+}
+
+enum TimestampType {
```

Comment:
Why do we need different types? OTLP only uses nanos in its proto. Please clarify the purpose.

Reply:
it's a holdover from bringing in the parquet gen field options. The generator defines a timestamp type to define how to serialize fields with parquet.

Reply:
Why do we need a "holdover" here? I don't understand.